application/views/qso/index.php

```diff
@@ -63,12 +63,12 @@
                       <!-- HTML for Date/Time -->
               <?php if ($this->session->userdata('user_qso_end_times')  == 1) { ?>
               <div class="row">
-                <div class="mb-3 col-md-3">
+                <div class="mb-3 col-sm-12 col-md-12 col-lg-4 col-xl-4 col-4">
                   <label for="start_date"><?= __("Date"); ?></label>
                   <input type="text" class="form-control form-control-sm input_date" name="start_date" id="start_date" tabindex="4" value="<?php echo date('d-m-Y'); ?>" <?php echo ($manual_mode == 0 ? "disabled" : "");  ?> required pattern="[0-3][0-9]-[0-1][0-9]-[0-9]{4}">
                 </div>
 
-                <div class="mb-3 col-md-4">
+                <div class="mb-3 col-sm-6 col-md-6 col-lg-4 col-xl-4 col-4">
```

Comment:
to gain a little bit more space for the time we could remove the padding here. Bootstrap comes with some margin so it would be fine. On Desktop and mobile

```suggestion
                <div class="mb-3 col-sm-6 col-md-6 col-lg-4 col-xl-4 col-4 ps-0 pe-0">
```
![image](https://github.com/user-attachments/assets/d573dcb4-31d0-44c6-af52-cc67cc2217e4)

![image](https://github.com/user-attachments/assets/a40f44dc-8ab4-4dcf-b789-94598bfd141f)

Reply:
Before:

<img width="445" alt="image" src="https://github.com/user-attachments/assets/6e7438ab-9464-40f3-bf94-c45e5de1e6e0">